.github/workflows/test_coverage.yml

```diff
@@ -35,9 +35,21 @@ jobs:
           python -m pip install --upgrade pip poetry==1.8.5
           python -m poetry install --with test,tutorials --all-extras --no-ansi --no-interaction
 
+      - name: Create gdf_account.json
+        uses: jsdaniell/create-json@v1.2.1
+        with:
+          name: "gdf_account.json"
+          json: ${{ secrets.GDF_ACCOUNT_JSON }}
+
+      - name: write realpath to env
+        run: |
+            echo "GDF_ACCOUNT_JSON=$(realpath ./gdf_account.json)" >> $GITHUB_ENV
+
```

Comment:
It's better to create this file as a temp file in the tests to avoid creating unnecessary and unrelated project-wide CI steps.